tox.ini

```diff
@@ -1,20 +1,11 @@
 [tox]
-envlist = py3{5,6}-{debug,release}
+envlist = py3{5,6,7}
 skip_missing_interpreters = True
-skipsdist = True
 
 [testenv]
 deps =
     -r{toxinidir}/requirements.txt
 
-basepython:
-    py35: python3.5
-    py36: python3.6
-
-setenv =
-    debug: PYTHONASYNCIODEBUG=x
-    release: PYTHONASYNCIODEBUG=
-
 commands =
     flake8 --show-source aioelasticsearch
```

Comment:
It's better to have separate envs for separate things

Reply:
fixed, returned it back